index.html

```diff
@@ -227,6 +227,28 @@ <h2>LETS CONNECT</h2>
             </div>
         </section>
     </section>
+    <section class="contact-form">
+        <div class="wrapper-contact">
+            <h2 class="contact">Contact me</h2>
+            <p class="contact-para">If you have an application you are interested in developing, a feature that you need
+                built or a project that needs coding.
+                I’d love to help with it!</p>
+            <form action="https://formspree.io/f/myyaqdnr" method="post">
+                <label for="name">
+                    <input type="text" name="name" placeholder="Enter your name" minlength="1" maxlength="30"
+                        required />
+                </label>
+                <label for="email">
+                    <input type="email" name="email" placeholder="Enter your email" required />
+                </label>
+                <label for="message" class="message-label">
+                    <textarea type="message" name="message" placeholder="Write your message here..." aria-multiline="1"
+                        maxlength="500" required></textarea>
+                </label>
```

Comment:
![Screenshot_2023-06-06_19-20-25](https://github.com/agneta1991/Portfolio/assets/120168752/ec0b63f4-1ea8-4a58-8ba7-1d98ac0e6b27)


@abdulhamiid i am not sure why on your device it opens as shown in the screenshot, on my vscode live server i use a toolbar toggle to change the screen size to a mobile screen and set dimensions as required of 375 and i build my code around that, there is a screenshot how it looks on my vs code live server

Reply:
aligns pretty well, also get in touch button is within a section and not as shown in your screenshot. could you advise me on how to avoid that sort of issues in the future?